frontend/src/metabase/components/EmptyState.jsx

```diff
@@ -29,7 +29,18 @@ const LegacyImage = props =>
       className={props.imageClassName}
     />
   ) : null;
-
+/**
+ * @typedef {Record<string, any>} EmptyStateProps
+ * @property {import("react").ReactNode} message
```

Comment:
Use `ReactNode` which would accept anything renderable e.g. boolean, number, string, ReactElement, etc.
```suggestion
 * @property {import("react").ReactNode} messager
```